internal/makefile/makefile.go

```diff
@@ -127,16 +127,6 @@ endif
 		prepareStaticRecipe = append(prepareStaticRecipe, "install-shellcheck")
 	}
 
-	if sr.UseGinkgo {
-		prepare.addRule(rule{
-			description: "Install ginkgo required when using it as test runner. This is used in CI before dropping privileges, you should probably install all the tools using your package manager",
-			phony:       true,
-			target:      "install-ginkgo",
```

Comment:
Do we have deps on `install-ginkgo` somewhere?

Reply:
To my knowledge, the `install-ginkgo` is currently only a dependency to the `build/cover.out` and `prepare-static-check` targets, both of which I've removed. So the only place where ginkgo is invoked would be during the `build/cover.out`, which now uses the `go run` approach I added.